CONTRIBUTING.md

```diff
@@ -106,6 +106,11 @@ If your language is not listed in Crowdin, please open an issue.
 ![Crowdin](https://d322cqt584bo4o.cloudfront.net/misskey/localized.svg)
 
 ## Development
+Before developing, you could set up Devcontainer with VSCode. Devcontainer has necessary setting for development server such as Redis and PostgreSQL.
+
+> [!NOTE]
+> Support from JetBrain's IDE may be limited. In doubtful situation, we recommend using VSCode.
+
```

Comment:
```suggestion
Before developing, you have to set up Redis and PostgreSQL.

This can be done with Devcontainer with VSCode. See below for devcontainer setup.

```

的な感じが適切だと思う

Reply:
done in https://github.com/misskey-dev/misskey/pull/14235/commits/0158f1615b23b376943f40355d45b41d956c1411